src/Scene/Quadtree.js

```diff
@@ -50,6 +50,9 @@ define('Scene/Quadtree', [
         this.tileType = type;
         this.minLevel = 2;
         this.maxLevel = 17;
+        this.wgs84TileLayer = null;
+        this.colorLayers = [];
+        this.elevationLayers = [];
```

Comment:
These attributes should be moved elsewhere (maybe at the globe level?), the quadtree might be used for other data types (point clouds, WFS data...).